src/i18n.ts

```diff
@@ -1,40 +1,65 @@
-import i18n from "i18next";
+import i18n, { InitOptions } from "i18next";
 import { initReactI18next } from "react-i18next";
 
-import Backend from "i18next-http-backend";
 import LanguageDetector from "i18next-browser-languagedetector";
-import { format as date_fnz_tz_format } from "date-fns-tz";
-import enCA from "date-fns/locale/en-CA";
-import frCA from "date-fns/locale/fr-CA";
-// don't want to use this?
-// have a look at the Quick start guide
-// for passing in lng and translations on init
+import { format as date_fns_tz_format } from "date-fns-tz";
+import {
+  arSA as date_fns_arSA,
+  enCA as date_fns_enCA,
+  frCA as date_fns_frCA,
+  pl as date_fns_pl,
+  ru as date_fns_ru,
+  zhCN as date_fns_zhCN,
+} from "date-fns/locale";
+import ar from "./locales/ar/translation.json";
+import en from "./locales/en/translation.json";
+import fr from "./locales/fr/translation.json";
+import pl from "./locales/pl/translation.json";
+import ru from "./locales/ru/translation.json";
+import zhCN from "./locales/zhCN/translation.json";
+
+export const i18nconfig: InitOptions = {
+  fallbackLng: "en-CA",
+  debug: false,
+  resources: {
+    ar: { translation: ar },
+    en: { translation: en },
+    fr: { translation: fr },
+    pl: { translation: pl },
+    ru: { translation: ru },
+    "zh-CN": { translation: zhCN },
+  },
+  interpolation: {
+    escapeValue: false, // not needed for react as it escapes by default
+    format: function format(value, fmt, lng) {
+      if (value instanceof Date && fmt !== undefined) {
+        const lang = lng?.substring(0, 2);
```

Comment:
Someone mentioned on the channel about supporting multiple Chinese languages - should we be evaluating the full string?

Reply:
There's two ways of writing Chinese - Simplified Chinese (zh-CN) and Traditional Chinese (zh-TW, zh-HK, zh-MO). Only Simplified Chinese is implemented for now because I haven't received translations for both. Date format currently being used is like 2021 5月 22 I think it's the same for both Simplified Chinese and Traditional Chinese (sorry I don't speak Chinese).

Reply:
Simplified Chinese is predominantly used in Mainland China and Traditional Chinese is predominantly used in Hong Kong/Macau/Taiwan.